front/pages/api/login.ts

```diff
@@ -444,7 +444,7 @@ export async function createAndLogMembership({
   workspace,
   role,
 }: {
-  user: UserType;
+  user: UserResource;
```

Comment:
cc @flvndvd do we want resources in lib/API signatures? I think no right?

Reply:
Having the resource in `lib` can be handy to avoid refetching it if actions need to be taken on the resource.

Reply:
So this means that we likely want to go all the way to returning the resource in auth.user() no?